ThirdParty/NimbleDroid.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  NimbleDroid.swift
+//  NimbleDroid
+//
+//  Created by Thomas Huzij on 5/8/17.
+//  Copyright © 2017 NimbleDroid. All rights reserved.
+//
+
+import Foundation
+
+public class NDScenario : NSObject {
+    static var coldStartup = false
+    static var initialized = false
+    static var scenario : String? = nil
+
+    public class func setup() {
```

Comment:
Why not make this a more traditional singleton? NDScenario.shared.foo(). Then you can get rid of all the `if !initialized` checks.

Reply:
We shouldn't touch this file directly, it is nimbledroid's.